rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -87,20 +87,11 @@ z_owned_keyexpr_t ros_topic_name_to_zenoh_key(const char * const topic_name, siz
   size_t topic_name_len = strlen(topic_name);
   size_t end_offset = topic_name_len;
 
-  if (topic_name_len > 0) {
-    if (topic_name[0] == '/') {
-      // Strip the leading '/'
-      start_offset = 1;
-    }
-    if (topic_name[end_offset - 1] == '/') {
-      // Strip the trailing '/'
-      end_offset -= 1;
-    }
-  }
+  std::string topic_nm = liveliness::mangle_name(topic_name);
```

Comment:
And update the call below with the new variable name.
```suggestion
  const std::string mangled_topic_name = liveliness::mangle_name(topic_name);
```